### PR TITLE
Feature/table component

### LINKED ISSUE #11: [FEATURE]: Create Table component
this component should be utils and can use in each sections from control panel

this component should have props with header columns `Array[]` and use data for render each rows

### PR DESCRIPTION
**Pull request by JoseCortezz25**
@Juanestban  closes #11

### CODE REVIEW

**Comment by Juanestban** [comment]
You should put in this description from pr something like this

**Example:**
# Issues
Closes #11

File: src/routes/routes.jsx [hunk 1]
```diff
@@ -20,6 +26,12 @@ const AppRoutes = () => {
             <Route index path="/aprender" element={<Plataform />} />
             <Route path="/aprender/:ruta" element={<Path />} />
             <Route path="/aprender/:ruta/:topic/:level" element={<Contents />}/>
+            <Route path="/admin" element={<Dashboard />}/> 
+            <Route path="/admin/usuarios" element={<Users />}/>
```
Comment (Juanestban) [inline src/routes/routes.jsx]: You could have a Array where have an 

- Component page
- path
- index `boolean | undefined`

File: src/pages/Admin/Syllabus/Syllabus.jsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+      <CoverGreetings greeting="Gestión del temario" isHome={false} />
+
+      <Link to="/admin/temario/añadir">
+        <button className="btnStandard btnBlue">
+          Crear nuevo temario
+        </button>
+      </Link>
+
+      <Table
+        columns={5}
```
Comment (Juanestban) [inline src/pages/Admin/Syllabus/Syllabus.jsx]: Remove this

File: src/pages/Admin/Resources/Resources.jsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+      />
+
+      <Link to="/admin/recurso/añadir">
+        <button className="btnStandard btnBlue">
+          Crear nuevo recurso recomendado
+        </button>
+      </Link>
+
+      <Table columns={4} headers={["Titulo", "Link", "Tipo", " Acciones"]}>
+        {RecommendedResources.map((resource) => (
```
Comment (Juanestban) [inline src/pages/Admin/Resources/Resources.jsx]: Try to desestucture this param

```suggestion
        {RecommendedResources.map(({ title, type, link, actions }) => (
```

File: src/pages/Admin/Resources/Resources.jsx [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+          <>
+            <li>{resource.title}</li>
```
Comment (Juanestban) [inline src/pages/Admin/Resources/Resources.jsx]: Try follow the same example

```suggestion
            <li>{title}</li>
```

File: src/pages/Admin/LearningPath/LearningPath.jsx [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+        columns={5}
+        headers={[
+          "Nombre de la ruta",
+          "Descripción",
+          "Link",
+          "Imagen",
+          "Acciones",
+        ]}
+      >
+        {LearningPaths.map((path) => (
```
Comment (Juanestban) [inline src/pages/Admin/LearningPath/LearningPath.jsx]: Same of above

File: src/pages/Admin/LearningPath/LearningPath.jsx [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+          <>
+            <li>{path.title}</li>
```
Comment (Juanestban) [inline src/pages/Admin/LearningPath/LearningPath.jsx]: Same of above

File: src/pages/Admin/Content/Content.jsx [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+          "Título",
+          "Descripción",
+          "Link",
+          "Nivel",
+          "Temario",
+          "Prioridad",
+          "Acciones",
+        ]}
+      >
+        {Contents.map((path) => (
```
Comment (Juanestban) [inline src/pages/Admin/Content/Content.jsx]: Same of above

File: src/pages/Admin/Content/Content.jsx [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+          <>
+            <li>{path.title}</li>
+            <li className="TextClipped">{path.description}</li>
+            <li className="TextClipped">{path.link}</li>
```
Comment (Juanestban) [inline src/pages/Admin/Content/Content.jsx]: Same of above

File: src/components/Table/Table.jsx [hunk 9]
```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import "./Table.css";
+
+const Table = ({ columns, headers, children }) => {
```
Comment (Juanestban) [inline src/components/Table/Table.jsx]: Remove prop `columns` and try use default params

```suggestion
const Table = ({ headers = [], children }) => {
```

**Review by Juanestban — CHANGES_REQUESTED** [review]
LGTM

Just you'll need make refactor

**Comment by JoseCortezz25** [comment]
Suggestions have been developed. @Juanestban

**Comment by Juanestban** [comment]
LGTM 💯 but remenber change the branch `master` because this should be merged to `dev` branch

**Review by Juanestban — APPROVED** [review]

**Comment by Juanestban** [comment]
and try to resolve the conflict  (don't worry, if you need help about this conflict, call me @Juanestban and I'll try to help you)

**Pull request merged**